todo-app/src/components/addingContainer.js

```diff
@@ -0,0 +1,43 @@
+
+import React, { Component } from "react";
+import FaPlus from "react-icons/lib/fa/plus"
+
+
+export default class AddingContainer extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            description: "",
+            deadLine: ""
+        }
+    }
+
+    handleDescriptionChange = (event) => {
+        this.setState({ description: event.target.value })
+    }
+    handleDeadlineChange = (event) => {
+        this.setState({ deadLine: event.target.value })
+
+    }
+    handleCreating = () => {
+        let descriptionInput = this.state.description
+        let deadLineInput = this.state.deadLine
+         this.props.create(descriptionInput, deadLineInput)
+    }
+
+    render() {
+        let description = <input type="text" value={this.state.description} onChange={this.handleDescriptionChange} />;
```

Comment:
First, you should be using `const` here instead of `let`, as this value is not being mutated.

Second, it would be more accurate to call these variables something like `descriptionInput` and `deadlineInput`, as you already have `description` and `deadline` as values in your state.